docs/integrations/ember-and-json-api.md

```diff
@@ -102,18 +118,16 @@ export default DS.JSONAPISerializer.extend({
 
 ```
 
+
 ## Including Nested Resources
```

Comment:
yeah, maybe mention the difference between including, and embedding, and that there are ways to embed, such as: https://github.com/NullVoxPopuli/aeonvera/blob/master/config/initializers/active_model_serializers.rb#L18